From: Christoph Egger <Christoph.Egger@gmx.de>
Subject: [PATCH] debian/correct-install

The variables setup for the install target does not work. We're correcting it here

Signed-off-by: Christoph Egger <Christoph.Egger@gmx.de>

---
 source/Irrlicht/Makefile |   13 +++++++++++--
 1 files changed, 11 insertions(+), 2 deletions(-)

diff --git a/source/Irrlicht/Makefile b/source/Irrlicht/Makefile
index 49f990d..a66980b 100644
--- a/source/Irrlicht/Makefile
+++ b/source/Irrlicht/Makefile
@@ -141,8 +141,17 @@ install install_osx:
 	mkdir -p $(INSTALL_DIR)/../include/irrlicht
 	cp ../../include/*.h $(INSTALL_DIR)/../include/irrlicht/
 	cp $(LIB_PATH)/$(SHARED_LIB).$(VERSION) $(INSTALL_DIR)
-	cd $(INSTALL_DIR) && ln -s -f $(SHARED_LIB).$(VERSION) $(SHARED_LIB)
-#	ldconfig -n $(INSTALL_DIR)
+	cp $(LIB_PATH)/$(STATIC_LIB) $(INSTALL_DIR)
+	cd $(INSTALL_DIR) && ln -s -f libIrrlicht.so.$(VERSION) $(SHARED_LIB)
+	#ldconfig -n $(INSTALL_DIR)
+
+install_osx:
+	$(RM) -r $(INSTALL_DIR)/../include/irrlicht
+	mkdir -p $(INSTALL_DIR)/../include/irrlicht
+	cp ../../include/*.h $(INSTALL_DIR)/../include/irrlicht
+	cp $(LIB_PATH)/$(SHARED_LIB).$(VERSION) $(INSTALL_DIR)
+	cd $(INSTALL_DIR) && ln -s libIrrlicht.dylib.$(VERSION) $(SHARED_LIB)
+	ldconfig -n $(INSTALL_DIR)
 
 TAGS:
 	ctags *.cpp ../../include/*.h *.h
-- 
tg: (9f40966..) debian/correct-install (depends on: master)
